src/components/ranking/RankingPeriodTabs.tsx

```diff
@@ -0,0 +1,43 @@
+import type { FC } from "react";
+import { Tabs, TabsList, TabsTrigger } from "@/components/ui/tabs.tsx";
+import type { Period } from "@/types/ranking.ts";
+
+type RankingPeriodTabsProps = {
+  selectedRankingPeriod: Period;
+  rankingPeriods: Period[];
+  onChangeRankingPeriod: (period: Period) => void;
+};
+
+const RankingPeriodTabs: FC<RankingPeriodTabsProps> = ({
+  selectedRankingPeriod,
+  rankingPeriods,
+  onChangeRankingPeriod,
+}) => {
+  return (
+    <Tabs
+      value={selectedRankingPeriod}
+      onValueChange={(value: string) => onChangeRankingPeriod(value as Period)}
+    >
+      <TabsList>
+        {rankingPeriods.map((period) => (
+          <TabsTrigger key={period} value={period}>
+            {period === "rt"
+              ? "실시간"
+              : period === "1d"
+                ? "1일"
+                : period === "7d"
+                  ? "1주일"
+                  : period === "1m"
+                    ? "1개월"
+                    : period === "3m"
+                      ? "3개월"
+                      : period === "6m"
+                        ? "6개월"
+                        : "1년"}
```

Comment:
이렇게 삼항 연산자가 여러 개 반복되는 경우에는 코드를 읽을 때, 어떤 케이스에 어떤 값이 되는지도 알기 어렵고 코드를 읽기도 어려워요.

이런 경우에는 아래와 같이 대응하면 좋아요.

1. 컴포넌트 바깥에 함수를 하나 만들어요. 그 함수의 이름은 `getPeriodText`와 같은 이름이면 좋겠네요.
2. 그 함수에서 `switch`문이나 `if-else`문을 활용하여 가독성 높은 코드로 만들어주세요.
3. 그리고 그 함수를 이곳에서 사용해주시면 됩니다.

```suggestion
            {getPeriodText(period)}
```